floodviz/map_utils.py

```diff
@@ -1,4 +1,7 @@
 import requests
+import json
+from string import Template
```

Comment:
According to PEP8 standard library imports should be grouped together and put before external package imports. So import requests should be moved below this one.